### PR TITLE
fix: add static-html.js to match Lit docs

### PR DESCRIPTION
**Pull request by wkillerud**
[`staticHtml`](https://github.com/finn-no/eik-shared-libs/blob/dd826a66032112af89339e7c1c7eb08fe78c8805/packages/lit/index.js#L27C17-L27C17) and `staticSvg` are non-obvious, and combining `html` from the main export with `unsafeStatic` breaks in unexpected ways, like `[object Object]` being rendered instead of the expected template result.

### CODE REVIEW

**Comment by wkillerud** [comment]
We'd then update the `lit/static-html.js` alias to point to this file instead.

How do we do this for `v2` as well @digitalsadhu ? Check out a previous tag and do some magic _release branch style_?

**Comment by digitalsadhu** [comment]
Lit produces 2 bundles for CDN:
* Core: https://cdn.jsdelivr.net/gh/lit/dist@3/core/lit-core.min.js
* All: https://cdn.jsdelivr.net/gh/lit/dist@3/all/lit-all.min.js

The `all` bundle includes `staticHtml`, `literal`, `staticSvg`, `unsafeStatic`, and `withStatic` and I think our bundle should if possible replicate the `all` bundle. We don't want to be relying on bundles on other CDNs and Lit does not expose the entry point file that produces the `all` bundle which is why we produce this one.  I have toyed with the idea of just changing the build step to copy their `all` bundle over onto Eik so that we stay in lock step. This might be a better way to go perhaps.

**Review by digitalsadhu — CHANGES_REQUESTED** [review]
So I think this is basically the way to go. There's a nice side win in your approach in that we split the core Lit bundle. 

Couple things though:
Firstly, I think we should go a step further and split it in 3. 1 for core, 1 for directives and 1 for static stuff. That will mean that anyone who is truly using the core of lit and nothing else won't pull in the rest. We may as well do that at the same time I think.

Secondly, it's really not easy to overwrite a package version on Eik. I can't do it, Trygve can but it involves manually deleting from the bucket as well as purging Fastly. Trygve is also on holiday. Best to avoid. We also can't just bump the version number since we stay in lock step with Lit's version. We only produce new versions, when the upstream lib produces a new version. What we can do is, is rename the package so thats an option. We could temporarily rename lit to lit-patch or whatever and then update the import map to point at this patch version. Once lit releases a new version, we could then rename back again. The downside of this approach is that while it would fix all new builds, it wouldn't fix code in prod. We could though just trigger a redeploy of each affected app to get the fix out.

**Comment by digitalsadhu** [comment]
The other advantage of a rename in the short term is that we can remove the deprecations and get the bundle size down since no one will automatically be upgrading to it without a build using the import map within which we control all the paths.

**Comment by wkillerud** [comment]
> Lit produces 2 bundles for CDN:
> 
> * Core: https://cdn.jsdelivr.net/gh/lit/dist@3/core/lit-core.min.js
> * All: https://cdn.jsdelivr.net/gh/lit/dist@3/all/lit-all.min.js
> 
> The `all` bundle includes `staticHtml`, `literal`, `staticSvg`, `unsafeStatic`, and `withStatic` and I think our bundle should if possible replicate the `all` bundle. 

Oh 😮 

> Does this mean we've found a bug in their setup here?

I'd say it's a lack of documentation on their part, then. There's [no mention of `staticHtml`](https://github.com/search?q=repo%3Alit%2Flit.dev+statichtml&type=code) anywhere in their docs.

I'm sure our root problem could be fixed by moving to `staticHtml` rather than `html`. The docs point folks to `import { html } from "lit/static-html.js"`, though. With our current setup, they don't get the version of `html` they think they do. I imagine this issue will keep popping up unless we change something.

File: packages/lit-2/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "version": "0.0.0",
+  "description": "An ESM version of Lit for Finn",
+  "type": "module",
+  "scripts": {
+    "copy": "echo 'noop';",
+    "build": "node ./esbuild.js",
+    "eik:login": "eik login",
+    "eik:publish": "eik publish",
+    "eik:alias": "eik npm-alias",
+    "eik:publish:ci": "../../scripts/publish.js lit lit"
```
Comment (wkillerud) [inline packages/lit-2/package.json:12]:
```suggestion
    "eik:publish:ci": "../../scripts/publish.js lit-2 lit"
```

File: packages/lit-3/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "version": "0.0.0",
+  "description": "An ESM version of Lit for Finn",
+  "type": "module",
+  "scripts": {
+    "copy": "echo 'noop';",
+    "build": "node ./esbuild.js",
+    "eik:login": "eik login",
+    "eik:publish": "eik publish",
+    "eik:alias": "eik npm-alias",
+    "eik:publish:ci": "../../scripts/publish.js lit lit"
```
Comment (wkillerud) [inline packages/lit-3/package.json:12]:
```suggestion
    "eik:publish:ci": "../../scripts/publish.js lit-3 lit"
```

**Review by digitalsadhu — APPROVED** [review]
LFGTM

**Pull request merged**